lib/BraveRewards.framework/Headers/BATBraveLedger.h

```diff
@@ -48,17 +48,24 @@ NS_SWIFT_NAME(BraveLedger)
 /// Creates a cryptocurrency wallet
 - (void)createWallet:(nullable void (^)(NSError * _Nullable error))completion;
 
-/// Fetch details about the users wallet (if they have one)
-- (void)fetchWalletDetails:(nullable void (^)(BATWalletProperties *))completion;
+/// Fetch details about the users wallet (if they have one) and assigns it to `walletInfo`
+- (void)fetchWalletDetails:(nullable void (^)(BATWalletProperties * _Nullable))completion;
 
 /// The users wallet info if one has been created
 @property (nonatomic, readonly, nullable) BATWalletProperties *walletInfo;
 
+/// Fetch details about the users wallet (if they have one) and assigns it to `balance`
+- (void)fetchBalance:(nullable void (^)(BATBalance * _Nullable))completion;
+
+/// The users current wallet balance and related info
+@property (nonatomic, readonly, nullable) BATBalance *balance;
+
 /// The wallet's passphrase. nil if the wallet has not been created yet
 @property (nonatomic, readonly, nullable) NSString *walletPassphrase;
 
 /// Recover the users wallet using their passphrase
-- (void)recoverWalletUsingPassphrase:(NSString *)passphrase completion:(nullable void (^)(NSError *_Nullable))completion;
+- (void)recoverWalletUsingPassphrase:(NSString *)passphrase
+                          completion:(nullable void (^)(NSError *_Nullable))completion;
```

Comment:
add space between `*` and `_Nullable`

Reply:
This will have to be done over in https://github.com/brave/brave-core/pull/2832